db/migrate/20150107044854_users_without_location_cleanup.rb

```diff
@@ -0,0 +1,9 @@
+class UsersWithoutLocationCleanup < ActiveRecord::Migration
+  def up
+    User.delete_all(location: [nil,''])
```

Comment:
@avinoth - I'm not sure, but I have a few ideas on how to get to it. 

Airbrake stores the parameters that are sent to the job (`user_id` in this case), we can use them to pick out the users and inspect them. We've made sure that there are no users with blank locations as of now - so this is most probably timing related (in between a user signing up and the location being set or something like that). If we can find a common trait among all the users that trigger this error, we should be able to get to the bottom of this issue. @seuros  - This would involve DB work, what do you think?